src/plugins/hypothesis/hypothesis.py

```diff
@@ -32,23 +32,49 @@ def process(self, uri, origin, scrape, last_check, event_dict):
             list: Contains results.
         """
 
-        api_url = 'https://hypothes.is/api/search'
-        parameters = {
+        events = {}
+
+        default_params = {
             'uri': uri.raw,
             'order': 'asc',
         }
         if last_check:
-            parameters.update(search_after=last_check.isoformat())
+            default_params.update(search_after=last_check.isoformat())
         
+        # 1) Process DOI
+        self.query_h(uri, origin, scrape, event_dict, events, default_params)
```

Comment:
I am sure there is a good reason for us to query the H.is API 3 times for the same URI. I just wonder if there is any way to group the search params into a single query (if the API supports `AND`-like logic) so we make 1 API call instead of 3.

Reply:
I can't find any documentation saying if/how much they limit requests to their API, but it does seem that we throw all the parameters into a single request it still returns the same results :+1: